app/containers/Modal.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { closeModal, modalSelectors } from 'reducers/modal'
+import { Modal as ModalOverlay } from 'components/UI'
+import Pay from 'containers/Pay'
+import Request from 'containers/Request'
+import Channels from 'containers/Channels'
+import ChannelCreateForm from 'containers/Channels/ChannelCreateForm'
+import ChannelCloseDialog from 'containers/Channels/ChannelCloseDialog'
+import ChannelDetail from 'containers/Channels/ChannelDetail'
+
+const ModalContent = ({ type, closeModal }) => {
+  switch (type) {
+    case 'PAY_FORM':
+      return <Pay width={9 / 16} mx="auto" />
+
+    case 'REQUEST_FORM':
+      return <Request width={9 / 16} mx="auto" />
+
+    case 'CHANNELS':
+      return <Channels mx={-4} />
+
+    case 'CHANNEL_CREATE_FORM':
+      return <ChannelCreateForm onSubmit={() => closeModal()} width={9 / 16} mx="auto" />
```

Comment:
All modals except for the last one have `mx="auto"` or them. I wonder what's the reason for that

Reply:
This is a bit of a strange case. For the, `CHANNEL_DETAIL` modal, the `ChannelDetail` component itself has the margin already applied because we want the content centered in the middle, but we want it's scrollbars to appear right at the edge of the modal rather than next to the centered content.